### PR TITLE
Add reverse nested bucket and moving function pipeline.

### PR DESCRIPTION
**Pull request by romainruaud**

### CODE REVIEW

File: src/module-elasticsuite-core/Search/Request/Aggregation/Pipeline/MovingFunction.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+     *
+     * @return string
+     */
+    public function getGapPolicy()
+    {
+        return $this->gapPolicy;
+    }
+
+    /**
+     * @return int
```
Comment (rbayet) [inline src/module-elasticsuite-core/Search/Request/Aggregation/Pipeline/MovingFunction.php]: Add function description ?

Comment (romainruaud) [reply src/module-elasticsuite-core/Search/Request/Aggregation/Pipeline/MovingFunction.php]: yes, something like "Get Pipeline Window" ? :D

**Comment by rbayet** [comment]
Otherwise, behaves as expected.

**Comment by romainruaud** [comment]
I updated the comments :)

**Pull request merged**